draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -9584,9 +9584,14 @@ Content-Encoding: gzip
    appropriate link.
 </t>
 <t>
-   After that is complete, please update the registry with the
+   After that is complete, please update the new registry with the
    header field names listed in the table of <xref target="field.names"/>.
 </t>
+<t>
+   Finally, please update the "Content-MD5" entry in the new registry to have
+   a status of 'obsoleted', and a reference of RFC7231, Appendix B, as this
+   instruction was omitted from that specification.
+</t>
 </section>
```

Comment:
change

 RFC7231, Appendix B

to

\<xref target="RFC7231" x:fmt="of" x:sec="B"/>